web/source/keymanweb.js

```diff
@@ -3811,9 +3802,7 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
     keymanweb._MutationAdditionObserved = function(Pelem) {
       if(!device.touchable) {
         // keymanweb.attachToControl is written to handle iframes, but setupDesktopElement is not.
-        if(keymanweb.setupDesktopElement(Pelem)) {
-          keymanweb.attachToControl(Pelem);
-        } else if(Pelem.tagName.toLowerCase() == 'iframe') {
+        if(Pelem.tagName.toLowerCase() == 'iframe') {
```

Comment:
Note that the only real change here is a re-ordering of the `if-else` construct.  The internal change to `setupDesktopElement` now returns `true` for IFrames, so we need our IFrame case to come first.

This is in prep for the changes in the next commit.